empathylink.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
+    settings = get_settings
+    @buddies = settings.hunting_buddies
+    @use_hodierna = DRC.get_settings.hashodierna
+    in_room = DRRoom.pcs
+    while true
+      in_room.each do | target |
+        if shouldheal(target)
+          touch(target)
+        end
+      end
+      pause 0.5
+      in_room = DRRoom.pcs
+    end
+  end
+
+  def populate_noheal_list
+    #UserVars.empathylink['xxxx'] = 2496928411
+    #UserVars.empathylink['xxxx'] = 2496928411
+  end
+
+  def shouldheal(target)
```

Comment:
What do you think about renaming this to `should_heal?` (note the `_` and `?`)?

Reply:
I can do that.